laurent22/joplin
# Desktop: Fixes #10672: External editing: Fix notes often not updated when saved from Vim

# Summary

This pull request fixes #10672 by removing an outdated workaround for [a resolved upstream issue](https://github.com/paulmillr/chokidar/issues/591).

# Testing plan

1. Set `gvim` as the external editor.
2. Open a note.
3. Click the "open in external editor" button.
4. Make a change.
5. Save with `:w`.
6. Verify that the change is visible in Joplin.
7. Repeat steps 4-6 for 2-3 more small changes. 
8. Set the external editor to `gnome-text-editor` (`/usr/bin/gnome-text-editor` for me).
9. Open two notes using the "toggle external editor" button.
10. Make changes to both notes from `gnome-text-editor` and save.
11. Verify that both changes are shown in Joplin.

This has been tested successfully on Ubuntu 24.04.
